third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -454,14 +458,12 @@ struct DotOpMFMAConversionHelper {
           Type elemTy = typeConverter->convertType(type);
           Type ty = vec_ty(elemTy, kBase);
           Value rawElems = tb.undef(ty);
-          for (int k = 0; k < kBase; ++k)
+          for (int k = 0; k < kBase; ++k) {
+            SmallVector<int64_t> indices = {b, nonK, kBaseVec, k};
```

Comment:
Note that constructing and destructing a `SmallVector` inside a nested loop like here can be expensive--you can either define the `indices` vector outside of the loop nest or maybe directly use initializer_list here.